src/sbtc-releases/sbtc-dev/get-started.md

```diff
@@ -0,0 +1,47 @@
+# Get Started with sBTC 0.1
+
+## Local setup
+### Requirements
+* install [clarinet](https://github.com/hirosystems/clarinet)
+* checkout main branch of repository [github.com/stacks-network/sbtc](https://github.com/stacks-network/sbtc)
+
+### Configuration
+1. Create configuration file `config.json` in a new folder `romeo/testing` with the following content:
+```
+{
+  "state_directory": "./state",
+  "contract": "../asset-contract/contracts/asset.clar",
+  "wif": "YOUR_WIF_CREDENTIALS",
```

Comment:
A nice follow-up would be to add docs on how to generate a wif from the sBTC-CLI. Ping @stjepangolemac.

Reply:
+1 to this, plus a very basic explainer of what a wif is for people unfamiliar, perhaps with a link to a more in-depth explanation

Reply:
Yeah maybe link to something similar to this https://en.bitcoin.it/wiki/Wallet_import_format. Also, there are multiple ways to generate credentials but we've made a tool specifically useful for sBTC work. It's the `sbtc-cli` package in the sbtc repo, sibling to `romeo`.

```
sbtc generate-from new
```

Look [here](https://github.com/stacks-network/sbtc/pull/109#pullrequestreview-1611412722) for example output.

Reply:
Done